src/components/Homepage/LanguageMorpher.tsx

```diff
@@ -0,0 +1,72 @@
+"use client"
+
+import Morpher from "@/components/Morpher"
+import { Button } from "@/components/ui/buttons/Button"
+
+import {
+  DESKTOP_LANGUAGE_BUTTON_NAME,
+  HAMBURGER_BUTTON_ID,
+  MOBILE_LANGUAGE_BUTTON_NAME,
+} from "@/lib/constants"
+
+import { useMediaQuery } from "@/hooks/useMediaQuery"
+
+const LanguageMorpher = () => {
+  const handleMobileClick = () => {
+    ;(document.getElementById(HAMBURGER_BUTTON_ID) as HTMLButtonElement).click()
+    setTimeout(
+      () =>
+        (
+          document.querySelector(
+            `button[name="${MOBILE_LANGUAGE_BUTTON_NAME}"`
+          ) as HTMLButtonElement
+        ).click(),
+      1
+    )
+  }
+  const handleDesktopClick = () => {
+    ;(
+      document.querySelector(
+        `button[name="${DESKTOP_LANGUAGE_BUTTON_NAME}"`
+      ) as HTMLButtonElement
+    ).click()
+  }
+
+  const [isLarge] = useMediaQuery(["(min-width: 48rem)"]) // TW md breakpoint, 768px
```

Comment:
FYI we could pull this value from https://github.com/ethereum/ethereum-org-website/blob/dev/src/lib/utils/screen.ts

Reply:
Thanks! I'll slip this refactor into the homepage refactor (#15553) since that's where this component is being used and I'm already in there.